src/tryGetDebouncedCompletions.ts

```diff
@@ -0,0 +1,61 @@
+import * as vscode from "vscode";
+import { Capability, isCapabilityEnabled } from "./capabilities/capabilities";
+import getInlineCompletionItems from "./getInlineCompletionItems";
+import TabnineInlineCompletionItem from "./inlineSuggestions/tabnineInlineCompletionItem";
+import { sleep, timed } from "./utils/utils";
+
+export default async function tryGetDebouncedCompletions(
+  document: vscode.TextDocument,
+  position: vscode.Position,
+  token: vscode.CancellationToken
+): Promise<
+  vscode.InlineCompletionList<TabnineInlineCompletionItem> | undefined
+> {
+  const { time, value: current } = await timed(() =>
+    getInlineCompletionItems(document, position)
+  );
+
+  const debounceTime = calculateDebounceValue(time);
+
+  if (debounceTime > 0) {
```

Comment:
i'd invert the condition for simplicity, because the `else` branch is only a `return` statement